x-pack/plugins/observability/public/utils/get_bucket_size/unit_to_seconds.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+const units = {
+  ms: 0.001,
+  s: 1,
+  m: 60,
+  h: 3600,
+  d: 86400,
+  w: 86400 * 7, // Hum... might be wrong
+  M: 86400 * 30, // this too... 29,30,31?
+  y: 86400 * 356, // Leap year?
```

Comment:
More like, skip year :D

```suggestion
  y: 86400 * 365, // Leap year?
```

Reply:
Hmm... I think we should avoid using the `getBucketSize` method from apm. It's custom, likely buggy and no tests. What do other teams do? 

@simianhacker I think you introduced me to the `getBucketSize` method. I can't find it anywhere in the infra code - so what do you do these days to get a bucket size for a given time range?

https://github.com/elastic/kibana/blob/e1a712e483deb2056e8de69e252dea3f18ea8d81/x-pack/plugins/apm/server/lib/helpers/get_bucket_size/index.ts#L13-L20

Reply:
@sqren Most of our charts that use `auto` or `>=1m` use TSVB's backend. I'm pretty sure the code I gave you originated in TSVB, which probably copied it and modified it from core. I doubt the quality is that bad and adding some tests should be pretty simple. 

Maybe we should move it to the Observability plugin and add some tests. It's an important function.

Reply:
> I doubt the quality is that bad and adding some tests should be pretty simple.

sorry for implying otherwise. When I searched through Kibana I got the impression it was abandonware and APM were the only consumers left.
If that's not the case we can keep this as-is.

Reply:
@simianhacker I added some tests https://github.com/cauemarcondes/kibana/blob/observability-overview/x-pack/plugins/observability/public/utils/get_bucket_size/index.test.ts#L9